bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/DeviceCapability.java

```diff
@@ -28,12 +28,15 @@
 public class DeviceCapability extends Capability {
     private static final int DATA_AGE_LIMIT_S = 3600;
 
+    protected boolean readonly = true;;
```

Comment:
Is it intentional that default value is the opposite default value of the API?

Reply:
Yes, it is because the first time the API will be call, I want to be sure that the request will not fail even for a favorite/guest station.
If you call the API with the device id of a favorite station and without setting get_favorites to true, the request fails.
Maybe I should rename this class member into `owned`.